### PR TITLE
Fix invite exception

### PR DESCRIPTION
**Pull request by binaryoverload**

### CODE REVIEW

File: src/main/java/stream/flarebot/flarebot/MessageUtils.java [hunk 1]
```diff
@@ -48,14 +49,24 @@ public static int getLength(EmbedBuilder embed) {
         return len;
     }
 
-    public static Message sendPM(User user, CharSequence message) {
-        return user.openPrivateChannel().complete()
-                .sendMessage(message.toString().substring(0, Math.min(message.length(), 1999))).complete();
+    public static Message sendPM(MessageChannel channel, User user, CharSequence message) {
```
Comment (WalshyDev) [inline src/main/java/stream/flarebot/flarebot/MessageUtils.java]: How come you're using CharSequence?

Comment (binaryoverload) [inline src/main/java/stream/flarebot/flarebot/MessageUtils.java]: That was in there originally, want me to change it?

**Comment by binaryoverload** [comment]
@WalshyDev Fixed

**Comment by binaryoverload** [comment]
yas! Thnx :P

File: src/main/java/stream/flarebot/flarebot/commands/Command.java [hunk 2]
```diff
@@ -21,7 +23,7 @@ default String getPermission() {
         return "flarebot." + getCommand();
     }
 
-    default Permission getDiscordPermission() {
+    default EnumSet<Permission> getDiscordPermission() {
         return null;
```
Comment (ArsenArsen) [inline src/main/java/stream/flarebot/flarebot/commands/Command.java]: `EnumSet.noneOf(Class)`

**Pull request merged**